router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -152,58 +158,14 @@ func createZipFile(filePath string, failedJobIds *[]int64, successJobIds *[]int6
 func (b *BingAdsBulkUploader) Upload(destination *backendconfig.DestinationT, asyncDestStruct *common.AsyncDestinationStruct) common.AsyncUploadOutput {
 
 	destConfig := DestinationConfig{}
-	jsonConfig, err := json.Marshal(destination.Config)
-	err = json.Unmarshal(jsonConfig, &destConfig)
+	jsonConfig, _ := json.Marshal(destination.Config)
+	_ = json.Unmarshal(jsonConfig, &destConfig)
 
 	failedJobIds := []int64{}
 	successJobIDs := []int64{}
 
 	filePath := createZipFile(asyncDestStruct.FileName, &failedJobIds, &successJobIDs, destConfig.AudienceId)
```

Comment:
&failedJobIds, &successJobIDs required?

Reply:
@ItsSudip

Reply:
We are handling the following scenario. While transforming `txt` to `CSV` file if we need to exclude some data due to size mismatch we add those jobs as failed jobs so that they will be retried.